example/src/main/java/eu/webeid/example/config/YAMLConfig.java

```diff
@@ -49,6 +58,22 @@ public String getLocalOrigin() {
     }
 
     public void setLocalOrigin(String localOrigin) {
+        if (StringUtils.endsWith(localOrigin, "/")) {
+            throw new IllegalArgumentException("Configuration parameter local-origin cannot end with '/': " + localOrigin);
+        }
+        if (StringUtils.startsWith(localOrigin, "http:")) {
```

Comment:
I feel uneasy about doing this so directly, folks may start copy-pasting this into their code without thinking.

Can we perhaps use a special profile `enable-http` and add thorough comments to make it absolutely clear that this should not be used in production?